generator-ui/src/scripts/background.js

```diff
@@ -53,18 +67,35 @@ chrome.commands.onCommand.addListener(async function (command) {
 
 /////////////////// HELPER FUNCTIONS ///////////////////
 
-async function displayNotification(title, message, icon) {
-    const notification = await chrome.notifications.create(
-        {
-            type: "basic",
-            iconUrl: icon,
-            title: title,
-            message: message,
-            silent: true
-        }
-    )
+// auto-inject content script on extension update
+function onInstallHandler(details) {
+    details.reason !== "install" && injectContentScript()
+};
+
+function injectContentScript() {
+    chrome.tabs.query({}, tabs => {
+        tabs.forEach(async (tab) => {
+          if (!tab.url.startsWith("chrome://") &&
```

Comment:
next commit will put excluded urls into a constant of type Set that you would just search for the tab url to see if you should inject

Reply:
done